index/views.py

```diff
@@ -48,7 +61,7 @@ def broken(request):
 
 def get_time(request):
     t = time.time()
-    tstr = datetime.datetime.fromtimestamp(t).strftime('%Y-%m-%d %H:%M:%S')
+    tstr = datetime.datetime.fromtimestamp(t).strftime('%Y-%m-%d %H:%M')
```

Comment:
You can keep the information for second.
Though the timer auto-update once a minute, users may refresh themselves.